docs/releases/release-8-3.md

```diff
@@ -0,0 +1,483 @@
+
+There’s a workaround: delete all EFI variables from the VM. In many cases, this is enough to recover. In some cases, this causes the loss of data needed to boot, requiring bootloader fixes with a virtual live CD/DVD. Debian, in particular, can be harder to recover, possibly due to its stricter application of UEFI standards.
+
+To delete the EFI variables of a VM (after a backup!), run this on a host in the pool: `xe vm-param-remove uuid=<vm-uuid> param-name=NVRAM param-key=EFI-variables`.
+
+You will also need to disable Secure Boot in the VM settings if it was enforced.
+
+### PBIS to winbind transition not automated for Active Directory settings
+
+See [Active Directory in XCP-ng](#active-directory-in-xcp-ng).
+
+## Known issues
+
+### IPv6-related issues
+
+With all the IPv4-only assumptions embedded in the code of various components, adding IPv6 support was a challenging task, but we managed to make it work. However, there are still some rough edges. We already have internal fixes for several of these issues, and they'll be released shortly.
+
+- Using IPv4 and IPv6 simultaneously ("dual stack") remains experimental and will be further stabilized through future updates.
+- When both IPv6 and a VLAN are configured for the management interface, the IP configuration may be lost after a reboot. A fix will be released shortly after the initial release.
+- Disabling port 80 doesn't work with IPv6 configurations on the management interface, as it currently only blocks IPv4 traffic. A fix will follow soon after the release.
+- Performing an emergency network restart reverts the configuration to IPv4 + DHCP instead of retaining IPv6 settings. This will also be fixed shortly after the release.
+- Ejecting a host from a pool causes it to fall back to IPv4 + DHCP, instead of maintaining its IPv6 configuration. A fix is coming soon.
+- DHCPv6 support is still incomplete.
+- Manually configured DNS settings can be overwritten by autoconf.
+
+### Known issues from XenServer 8
+
+Though not all issues from XenServer 8 apply to XCP-ng 8.3, some of the information and tips provided in [XenServer 8's Known Issues](https://docs.xenserver.com/en-us/xenserver/8/whats-new/known-issues) might still be useful for certain scenarios.
+
+### Cross-pool live migration from XenServer < 7.1
+
+Live migrating a VM from an older XenServer version can sometimes result in errors, leading to the following consequences:
+* The VM may reboot.
+* The VM may get duplicated, meaning the same VM UUID (and usually its VDIs too) will appear both on the sender and receiver host. In such a case, you'll need to remove the duplicate from the receiver host.
+
+It would require a hotfix to the old XenServer, but since those versions are no longer supported, XenServer won't be developing one.
```

Comment:
Tell that Xen Orchestra warm migration is the solution (you can link https://xen-orchestra.com/blog/warm-migration-with-xen-orchestra/)